docs/installation.rst

```diff
@@ -260,6 +260,12 @@ macFUSE, which is available via `github
 
     brew install --cask osxfuse
 
+When installing Borg via ``pip``, be sure to use ``pip install borgbackup[llfuse]``,
+since macFUSE only supports FUSE API v2.
+
+Since Homebrew won't link the installed ``openssl`` formula, set the ``BORG_OPENSSL_PREFIX`` 
+environment variable to ``/usr/local/Cellar/openssl`` before running ``pip install``.
```

Comment:
is there no `@1.1` for this case?

Reply:
It works without, but I'm not sure why.

Make it `@X.X` to cover future version changes?

Reply:
I don't know any specifics, i just remembered from other occasions that there often used to be a `@1.1` appended to the openssl path. not sure what the path without points to, if there is such a path.

can you check on your system whether the path exists / whether there are symlinks?

Reply:
Already checked. Only the `@1.1` path is there. No symlink. Inside it is a more specific version folder.

So there is already some flexibility finding the folder. Maybe that's why it works. since this line in `setup_crypto.py` is at least one folder level off too: `lib_dir = os.path.join(system_prefix, 'lib')`

```
$ ls -l /usr/local/Cellar/ | grep openssl
drwxr-xr-x  3 manu  staff  96 Jan 13 15:33 openssl@1.1
```

```
$ tree -L 2 /usr/local/Cellar/openssl@1.1
/usr/local/Cellar/openssl@1.1
└── 1.1.1i
    ├── AUTHORS
    ├── CHANGES
    ├── INSTALL_RECEIPT.json
    ├── LICENSE
    ├── NEWS
    ├── README
    ├── bin
    ├── include
    ├── lib
    └── share

5 directories, 6 files
```

---

On the other hand, the pkg-config folder without version is  real symlink. Maybe we use that instead of `BORG_OPENSSL_PREFIX `?

```
$ ls -l /usr/local/opt/ | grep openssl
lrwxr-xr-x  1 manu  admin  28 Jan 13 11:48 openssl -> ../Cellar/openssl@1.1/1.1.1i
lrwxr-xr-x  1 manu  admin  28 Jan 13 11:48 openssl@1.1 -> ../Cellar/openssl@1.1/1.1.1i
```

Would be this: `PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig`

Reply:
guess if we want 1.1, we should use the @1.1 version.
Some people might have 1.0 still available and there might be some newer openssl version in the future.

if the pkgconfig way can be made working, i guess we should use that, pkgconfig's job is to make it easy to find headers/libs/etc.

Reply:
In that case, we should point it to the pkgconfig path of openssl 1.1 specifically:

![Screen Shot 2021-02-17 at 08 44 25](https://user-images.githubusercontent.com/3916435/108140103-5da38880-70fc-11eb-84f7-48715ad14050.jpg)

Any style preference on how to write this? Show it together with the `pip` command?

Reply:
How about:
```
When installing Borg via ``pip``, be sure to install the ``llfuse`` extra,
since macFUSE only supports FUSE API v2. Also, since Homebrew won't link
the installed ``openssl`` formula, point pkg-config to the correct path:

    PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig" pip install borgbackup[llfuse]
```

Reply:
Works too and it's all in one. 😄

Reply:
Updated PR